src/mainwindow.cpp

```diff
@@ -1011,15 +1014,15 @@ void MainWindow::onDotsButtonClicked()
     viewMenu->setToolTipsVisible(true);
     mainMenu.setToolTipsVisible(true);
 
-    mainMenu.setStyleSheet(QStringLiteral(
+    /*mainMenu.setStyleSheet(QStringLiteral(
                                "QMenu { "
                                "  background-color: rgb(255, 255, 255); "
                                "  border: 1px solid #C7C7C7; "
                                "  }"
                                "QMenu::item:selected { "
                                "  background: 1px solid #308CC6; "
                                "}")
-                           );
+                           );*/
```

Comment:
Yes, I think this is also an improvement on Linux where the popup looked a bit broken with this stylesheet as well:

![image](https://user-images.githubusercontent.com/531764/87817580-7f3c4900-c869-11ea-9b4f-0a9bb93f6bb7.png)

The checkmark is too small, and it looks better without stylesheet:

![image](https://user-images.githubusercontent.com/531764/87817633-98dd9080-c869-11ea-8496-ecc18ea1fcbf.png)

It will depend on the platform style, but I think that's a good thing.

I would only suggest that you remove these lines, instead of commenting them out.

Reply:
Will do!